src/popup.html

```diff
@@ -56,723 +50,123 @@ <h1 class="mdc-top-app-bar__title">AO3 Podfic Posting Helper</h1>
       <form id="form">
         <div class="list" role="group">
           <div class="form-item">
-            <label
-              class="mdc-text-field mdc-text-field--filled mdc-theme--primary url-text-field">
-              <span class="mdc-text-field__ripple"></span>
-              <span class="mdc-floating-label" id="url-label">
-                URL of work to import from
-              </span>
-              <input
-                class="mdc-text-field__input"
-                type="url"
-                aria-labelledby="url-label"
-                id="url-input"
-                required
-                aria-controls="url-error-hint"
-                aria-describedby="url-error-hint"
-                pattern="https://archiveofourown.org/(.*/)?works/[0-9]+.*"
-                title="Must be an AO3 work URL"
-                name="ao3-url"
-                spellcheck="false" />
-              <span class="mdc-line-ripple"></span>
-            </label>
-            <div class="mdc-text-field-helper-line">
-              <div
-                class="mdc-text-field-helper-text mdc-text-field-helper-text--validation-msg"
-                id="url-error-hint"
-                aria-hidden="true"></div>
-            </div>
+            <md-filled-text-field
+              id="url-input"
+              label="URL of work to import from"
+              type="url"
+              pattern="https://archiveofourown.org/(.*/)?works/[0-9]+.*"
```

Comment:
I would suggest changing this regex to support urls like https://archiveofourown.org/chapters/134876962 but tbh I'm scared to find out if that has special redirect rules so I guess let's not for now

(If I were suggesting changing it, maybe to something like this?:)
`https://archiveofourown.org/(.*/)?(works|chapters)/[0-9]+.*`